src/factory.js

```diff
@@ -179,6 +179,14 @@ module.exports = ({ Component, createElement, PropTypes }) => {
             state = initialState(props);
 
             keys(state).forEach(k => {
+              if (
+                computed !== undefined &&
+                Object.prototype.hasOwnProperty.call(computed, k)
+              ) {
+                throw new TypeError(
+                  `conflict: "${k}" is defined both in state and computed`
+                );
+              }
               if (!(k in stateDescriptors)) {
```

Comment:
Why did I do this test?

Reply:
I think it's avoid state entries to shadow computed, but I think it's no longer necessary with your test.